### PR TITLE
Update US22 Unique IDs for restructured file

### PR DESCRIPTION
**Pull request by 55Brian55**
Updated the GEDCOM Parser to first check if ID is in dictionary and print Error if it is. Added Unit test to test that family & individual dictionaries are loaded with the correct '0' level IDs.

### CODE REVIEW

File: GedcomParser.py [hunk 1]
```diff
@@ -70,10 +69,16 @@ def parse_valid_results(results):
     results_iter = iter(results)
     for result in results_iter:
         if result[1] == 'FAM':
-            families[result[3]] = Family(result[3])
+            if result[3] in families.keys():        # family id is a duplicate
```
Comment (CCorrado) [inline GedcomParser.py]: I'm not sure if this belongs here. This method is collecting all of the data from the .GED files and storing them in the dictionaries.

We should only be performing post-processing on the data in the files that already exist, that way we don't trip over each other in each user story.

**Comment by 55Brian55** [comment]
Hi Chris,
As we discussed previously the dictionary will not allow duplicates but will overwrite duplicate entries. Having the functionality here preserves the first entry into the dictionary as the correct one. Let’s discuss this morning.

**Comment by CCorrado** [comment]
--- Requirements ---
GEDCOM test output file
The output (see below for an example) should include:
1. Summary, with headers, of all individuals 
2. Summary,with headers, of all families
3. Demonstration of user stories from current and all previous sprints. 

This change will break use cases 1 and 2, by not printing out all families and individuals.

**Comment by CCorrado** [comment]
Based on our conversation, the first ID for any individual or family will be the one that gets printed in the summary table. Any duplicate will get printed out as an error string.

**Pull request closed without merging**